.github/workflows/build-lint-test.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: Node.js CI - Build + Lint + Test
+
+on: pull_request
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Use Node.js v16
+              uses: actions/setup-node@v2
+              with:
+                  node-version: '16.x'
+                  cache: 'npm'
+
+            - name: 📦 Install dependencies
+              run: npm ci
+
+            - name: 🎨 Run ESLint on JS/TS files
+              run: npm run lint
+
+            - name: 🏗️ Build the lib
+              run: npm run build
+
+            - name: 🧪 test the build
+              run: npm run test
+
+    test-suites:
+        strategy:
+            matrix:
+                package:
+                    ['helpers', 'IPFS', 'LSP2', 'LSP3', 'LSP4', 'LSP5', 'LSP6', 'LSP12', 'LSP23']
```

Comment:
I don't think this matrix for the `test-suites` is necessary, since the `npm run test` command above in the previous step is going to run the tests from all the tests file. It seems redundant to me:

<img width="627" alt="image" src="https://github.com/lukso-network/lsp-utils/assets/31145285/23e16761-4b77-4ef7-9c3f-6bc8c7925897">